### ISSUE TITLE
Use line breaks after each verse setting missing in 3.0

### ISSUE THREAD

**Issue by 2resinjn5** [bibletime/bibletime#285]
Version 2.11.2 on Mac OS X has an option "Use line breaks after each verse". Version 3.0 removed this option. Can it be restored? I think it is important to read the bible in paragraphs.

**Comment by gholmlund** [comment]
It was taken out for a reason. Its implementation is not possible with the new continuous scrolling feature which allows you to move from chapter to chapter just by scrolling. I think this is a much more important feature.

**Comment by 2resinjn5** [comment]
"And Bible" added continuous scrolling without losing the paragraph feature. I find much meaning of the text is lost without paragraphs.

I wonder whether it could be either one or the other?

As it stands I resort to running Xiphos under WINE on a Mac. Of course, if it is only me in the world who feels this is important, BibleTime 2.11 still works in my case because I will never update beyond Mac OS Sierra.

**Comment by adamoell** [comment]
I would agree that removing paragraph formatting is a significant retrograde step. It makes reading text in Bibletime feel stilted and awkward by comparison to the alternatives.

**Comment by adamoell** [comment]
After a couple of days I ended up rolling back to 2.11.2. Linebreaks after every verse breaks up the flow of reading and studying; for me, proper paragraph formatting is vastly more important for reading and study than chapter-to-chapter scrolling.

**Comment by jaakristioja** [comment]
I too think this is important. I don't think it is entirely impossible to do this.

**Comment by gholmlund** [comment]
I agree. I will look into it.

**Comment by adamoell** [comment]
That's brilliant, gentlemen, thank you so much!

**Comment by doonoo** [comment]
Any progress?

**Comment by jaakristioja** [comment]
We will not implement this for the stable 3.0 series of BibleTime, but we have set the milestone to 3.1.

**Comment by kalemas** [comment]
i was thinking about this in sense of BibleTime Mini, and decided that underlying model simply should have per chapter rendering mode. Parallel mode should also map texts on per chapter basis.

**Comment by gholmlund** [comment]
I agree that chapter rendering would fix this issue. I have not been able to figure out how to position the text to places within the chapter (verses other than verse 1).
